test/dotnet-new.Tests/Approvals/DotnetNewHelpTests.CanShowHelpForTemplate_MatchOnLanguage.verified.txt

```diff
@@ -16,16 +16,16 @@ Options:
   --type <project>        Specifies the template type to instantiate.
 
 Template options:
-  -f, --framework <net6.0|net7.0|net8.0|net9.0>  The target framework for the project.
-                                                 Type: choice
-                                                   net9.0  Target net9.0
-                                                   net8.0  Target net8.0
-                                                   net7.0  Target net7.0
-                                                   net6.0  Target net6.0
-                                                 Default: net9.0
-  --no-restore                                   If specified, skips the automatic restore of the project on create.
-                                                 Type: bool
-                                                 Default: false
+  -f, --framework <net10.0|net6.0|net7.0|net8.0>  The target framework for the project.
```

Comment:
Why is net9 missing from this list? Shouldn't that show up if 9.0 is installed? Is that missing somewhere from the test setup (I see it in the restoretoolset scripts so not sure)?

Reply:
tested locally, we only show the net version when those templates are installed (presumably by another SDK or by VS). I think the build machines we're building with are still 17.11 versions so no .net9 templates installed yet so this is ok for now but will break once the VS images get updated.